frontend/web/js/api.ts

```diff
@@ -211,9 +212,19 @@ export class ApiService {
     );
   };
 
-  getSampleGroupMemberships = async (sampleIds: string[], signal?: AbortSignal) => {
-    return this.http.request<SampleGroupMembership>(
-      `/samples/group-memberships?${objectToQueryParams({ ids: sampleIds })}`,
+  getMembershipByGroups = async (groupIds: string[], signal?: AbortSignal) => {
```

Comment:
Maybe I am confused, but is it `getMembershipByGroups` or `getMembershipsByGroups` here